config/manifests/bases/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     - description: OpenTelemetryCollector is the Schema for the opentelemetrycollectors
         API
       kind: OpenTelemetryCollector
-      name: opentelemetrycollectors.core.opentelemetry.io
+      name: opentelemetrycollectors.opentelemetry.io
```

Comment:
Not sure this is a breaking change, is it? If so, I'll revert this. I want to avoid breaking current users if possible at all, and this change wouldn't be justifiable.